campaigns/fhv/10_the_second_evening.json

```diff
@@ -1436,7 +1436,11 @@
         ]
       }
     },
-
+    {
+      "id": "helen_peters_joined_the_survey_story",
```

Comment:
don't know where the narrations are to know what to reference and whether it exists